src/dataflow/src/sink/kafka.rs

```diff
@@ -898,7 +906,10 @@ where
                         }
                     }
                     SendState::Shutdown => {
-                        s.shutdown_flag.store(false, Ordering::SeqCst);
+                        s.shutdown_flag.store(true, Ordering::SeqCst);
+                        // Indicate that the sink is closed to everyone else who
+                        // might be tracking its write frontier.
+                        info!("shutting down kafka sink: {}", &s.name);
```

Comment:
nit: we now log twice.

Reply:
yeah - i felt that was defensible as the other log site is a debug log and could be useful to know how many times we hit the sink operator even after its shutdown flag is set. Previously, we had an info log that could repeat indefinitely, so this seems better?

Reply:
Makes sense!